### ISSUE TITLE
GUI MessageWindow will  no more appear in 0.9.8

### ISSUE THREAD

**Issue by headmin** [google/santa#41]
We run Santa connected to Zentral (https://github.com/zentralopensource) as TLS server for Santa - while base functionality is same, blacklisting works as with santa 0.9.7 - we don't see any GUI MessageWindow on clients with Santa 0.9.8. 
Once we reinstall previous santa 0.9.7 version again via munki the GUI window appears as did in all earlier versions of Santa we have used. 
Would be great to get a short hint what could go wrong, new key required ? - probably we just missed something from release notes for Santa 0.9.8 ?

BTW the JSON blob that works with 0.9.7 and fails with 0.9.8:
```{'custom_msg': 'Blocked to launch Transmission.app - be aware this is is a infected binary',
 'policy': 'BLACKLIST',
 'rule_type': 'BINARY',
 'sha256': 'ddc3dbee2a8ea9d8ed93f0843400653a89350612f2914868485476a847c6484a'}```

**Comment by russellhancox** [comment]
Is the binary being blocked correctly but the notification is not appearing or is the binary not being blocked either?

**Comment by headmin** [comment]
sure the binary is blocked santa works as should be - it's just the `custom_msg` not  displayed to the user

**Comment by russellhancox** [comment]
OK, nothing has changed in the sync code as far as that's concerned but the GUI code changed a bit. I think what's probably happening is the GUI is running but not connected properly.

What happens if you `killall Santa` and try running a blocked binary, do you then get the popup appear?

**Comment by russellhancox** [comment]
After more testing I'm fairly sure this is caused by 736b45bb46e852e9688c60d35ed215e681715e72 as connections aren't being invalidated properly.

**Comment by russellhancox** [comment]
Fixed in 916c3c7a2aee39b58b0813a4ab465de335eb312d, released in [0.9.9](https://github.com/google/santa/releases/0.9.9)

**Comment by headmin** [comment]
thanks installed on my machine things seem to work, will test in our fleet tomorrow
